README.md

```diff
@@ -1,2 +1,31 @@
 # flight-booking-api
 api for booking flight tickets
+
+
+### Testing
+
+to test integration tests:
+```
+make testintegration
+```
+Before testing migrate up testing database by command:
+```
+make migrate-up-test
+```
+
+also set config variables as below example:
+
+```
+POSTGRES_HOST=localhost
```

Comment:
You can make an `.env` file for local testing or have a make command to set them before launcking tests